src/rtkpos.c

```diff
@@ -1900,9 +1900,9 @@ static int relpos(rtk_t *rtk, const obsd_t *obs, int nu, int nr,
         }
     }
     /* resolve integer ambiguity by WL */
-    else if (stat!=SOLQ_NONE&&rtk->opt.modear==ARMODE_WL) {
+    else if (stat!=SOLQ_NONE&&(rtk->opt.modear==ARMODE_WL||rtk->opt.modear==ARMODE_WL_FIXHOLD)) {
```

Comment:
rtk->opt.modear==ARMODE_WL_FIXHOLD

Reply:
There is a check 2 lines later for `if(rtk->opt.modear==ARMODE_WL_FIXHOLD)`.  If we don't check for `ARMODE_WL_FIXHOLD` here, then the line 2 lines below will never evaluate as true...

Reply:
ah, yeah, maybe we should define a separate variable as `opt.modear_hold`? So that we could only check `if(rtk->opt.modear==ARMODE_WL)` is true to decide whether to do widelane fixing or not, and then check `if(rtk->opt.modear_hold==ARMODE_WL_FIXHOLD)` is true or not to decide to call `holdamb` function.

Reply:
If we are going to take that approach then I think it would also make sense to change the `ARMODE_INST` mode to use the new `modear_hold` variable as well.

Reply:
probably we don't need to change that. Any `ARMODE` other than `WL/WLNL/TCAR` fixing will go into `resamb_LAMBDA`, and if it is ARMODE_FIXHOLD, then holdamd function is called.